### PR TITLE
feat: add season translation

### PR DESCRIPTION
**Pull request by just-tktok**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tv_seasons.go [hunk 1]
```diff
@@ -324,3 +345,29 @@ func (c *Client) GetTVSeasonVideos(
...
+		"%s%s%d%s%d/translations?api_key=%s%s",
+		baseURL,
+		tvURL,
+		id,
+		tvSeasonURL,
+		seasonNumber,
+		c.apiKey,
+		options,
+	)
+	tvSeasonVideos := TVSeasonTranslations{}
```
[the remark this answers is not in the document]
Comment (just-tktok) [reply tv_seasons.go:368]: ok

**Comment by just-tktok** [comment]
@cyruzin i fix it

**Comment by cyruzin** [comment]
> @cyruzin i fix it

Hi, the test is failing:

./tv_seasons.go:357: Sprintf format %s reads arg #7, but call has 6 args

**Comment by just-tktok** [comment]
> > @cyruzin i fix it
> 
> Hi, the test is failing:
> 
> ./tv_seasons.go:357: Sprintf format %s reads arg #7, but call has 6 args

fix it

**Comment by cyruzin** [comment]
> > > @cyruzin i fix it
> > 
> > 
> > Hi, the test is failing:
> > ./tv_seasons.go:357: Sprintf format %s reads arg #7, but call has 6 args
> 
> fix it

thanks

**Pull request merged**